### PR TITLE
changed the 'except E, e:' style to be 'except as e:' ...

### PR DESCRIPTION
**Pull request by japrogramer**
since the former is deprecated.
Other changes remove trailing spaces and lines with just spaces.

### CODE REVIEW

File: lang/py/src/avro/ipc.py [hunk 1]
```diff
@@ -296,9 +296,9 @@ def respond(self, call_request):
       # perform server logic
       try:
         response = self.invoke(local_message, request)
-      except AvroRemoteException, e:
+      except AvroRemotelxception as e:
```
Comment (aukevanleeuwen) [inline lang/py/src/avro/ipc.py]: Definitely no Python guru here, so don't consider this an actual review, but this is most likely a typo: `AvroRemotelxception`.

Comment (japrogramer) [reply lang/py/src/avro/ipc.py]: Thanks didn't catch that one, Ill update my pull request

**Comment by spacharya** [comment]
Thanks @japrogramer  for the changes
And @aukevanleeuwen  for the initial review. If no one has any concerns I will merge them in the next few days.

**Comment by japrogramer** [comment]
@spacharya  @aukevanleeuwen grate, thanks for the reviews

**Comment by bogdanalov** [comment]
Hello, guys! I am getting error when I try to import it from python2.7 . The whole module cannot be imported. Python version: 2.7.13. Also I'm trying on Jenkins with python2.7 and still fails Jenkins Python version: 2.7.6

**Comment by delusionxb** [comment]
it seems there are unsupported syntax within avro 1.8.2 + python 3.6

https://docs.python.org/3/whatsnew/3.0.html
datafile.py, line 20 -23 should be
try:
  from io import BytesIO
except ImportError:
  from io import StringIO

BUT still invalid syntax after this fix
avro/io.py", line 200
    bits = (((ord(self.read(1)) & 0xffL)) |
                                      ^
SyntaxError: invalid syntax

so... plz take a complete go through

File: lang/py/src/avro/schema.py [hunk 2]
```diff
@@ -773,7 +773,7 @@ def parse(json_string):
   # parse the JSON
   try:
     json_data = json.loads(json_string)
-  except Exception, e:
+  except Exception as e:
     import sys
     raise SchemaParseException('Error parsing JSON: %s, error = %s'
                                % (json_string, e)), None, sys.exc_info()[2]
```
[the remark this answers is not in the document]
Comment (japrogramer) [reply lang/py/src/avro/schema.py]: python, changed the way exceptions are raised. http://legacy.python.org/dev/peps/pep-3109/#grammar-changes

[the remark this answers is not in the document]
Comment (delusionxb) [reply lang/py/src/avro/schema.py]: these 'invalid syntax' make me think the avro guys shipped their cargo completed for python2

[the remark this answers is not in the document]
Comment (spacharya) [reply lang/py/src/avro/schema.py]: Hi, @japrogramer and @delusionxb.
Your feedback is very much appreciated.
However, seems like you'll are testing python3 on Avro python-2 library.
You can see the source-code for Avro-python 3 here: https://github.com/apache/avro/tree/master/lang/py3
The python 2 library is: https://github.com/apache/avro/tree/master/lang/py
Hope this helps the confusion you'll are having.

[the remark this answers is not in the document]
Comment (japrogramer) [reply lang/py/src/avro/schema.py]: @spacharya I noticed that soon after replying to @delusionxb  comment.

[the remark this answers is not in the document]
Comment (delusionxb) [reply lang/py/src/avro/schema.py]: @spacharya  Well, you are correct about that... but I grabbed 1.8.2 from 'pip install', should this be considered the 'official' release? I mean I don't think I need to (or should) use it by importing the source code, right? Correct me on this concept if I'm wrong, thanks a lot.

[the remark this answers is not in the document]
Comment (delusionxb) [reply lang/py/src/avro/schema.py]: I just noticed that the current version is '1.9.0-SNAPSHOT', is this the reason 1.8.2 fails? Does this mean avro installed by 'pip install' will be v1.9.0 in the futruen and those 'invalid syntax' will be gone?

**Comment by soxofaan** [comment]
As far as I understand there is no Apache Avro Python library that installs both python 2 and 3.
- For python 2 you have to (pip) install `avro`
- For python 3 you have to (pip) install `avro-python3`

I'm using the latter, without the kind of syntax issues mentioned above.

I think this PR should be canceled and closed

**Pull request closed without merging**